src/components/Overlays/Modal/InformationModal/index.tsx

```diff
@@ -19,11 +19,13 @@ export const InformationModal = ({
   disableBackdrop
 }: InformationModalProps) => (
   <Modal isOpen={isOpen} title={title} hideModal={hideModal} disableBackdrop={disableBackdrop}>
-    <ModalHeader>
-      <ModalHeaderContent className="non-dragable">
-        {title} <IconButton name="Close" backgroundGlowOnHover onClick={hideModal} />
-      </ModalHeaderContent>
-    </ModalHeader>
+    <Styles.ModalHeader>
+      <Styles.ModalHeaderContent className="non-dragable">
```

Comment:
Remove the `non-dragable` class, because:
1. You shouldn't be using raw classes.
2. Your class defines the browser variants which is redundant bc the CSS-in-JS libraries do this for you.
3. Just add `user-select: none` to the styled components.

```suggestion
      <Styles.ModalHeaderContent>
```

Reply:
Resolved